app/src/main/res/values/strings.xml

```diff
@@ -712,4 +712,7 @@
 
   <!-- Teleconsult Prescription Screen -->
   <string name="screen_teleconsult_prescription_patient_details" translatable="false">%1$s, %2$s, %3$s</string>
-</resources>
+
+  <!-- Teleconsult Patient Info View -->
+  <string name="view_teleconsult_patient_info" translatable="false">%1$s, %2$s, %3$s</string>
```

Comment:
Something to think about for the future, but we have a lot of untranslatable strings now, and I'm wondering if we should extract those out into a separate string resources file for better separation at some point.

Reply:
Definitely, it's also worth considering using a single string for commonly used text like this one.

Reply:
Can we raise a ticket for this so we can get to it at some point?

Reply:
Tracking stories here:
- https://app.clubhouse.io/simpledotorg/story/1236/extract-non-translatable-strings-into-separate-string-resource-file-for-better-separation
- https://app.clubhouse.io/simpledotorg/story/1237/extract-common-strings-into-a-single-string